front/lib/api/assistant/configuration.ts

```diff
@@ -56,13 +57,13 @@ import { generateModelSId } from "@app/lib/utils";
  * Get an agent configuration
  *
  */
-export async function getAgentConfiguration(
+export async function getAgentConfigurationDetailedView(
   auth: Authenticator,
   agentId: string,
   preFetchedAgentConfiguration?: AgentConfiguration,
   preFetchedUserRelation?: AgentUserRelation | null,
   preFetchedDataSourceConfigurations?: AgentDataSourceConfiguration[]
```

Comment:
We probably don't need these anymore no?

Reply:
At least I would expect the prefetch to not be needed because as we list we don't call that one anymore?

Reply:
Yep I don't think we need the prefetch 👍